sway-core/src/control_flow_analysis/dead_code_analysis.rs

```diff
@@ -1049,6 +1028,41 @@ fn connect_expression(
     }
 }
 
+fn connect_intrinsic_function(
+    kind: &TypedIntrinsicFunctionKind,
+    graph: &mut ControlFlowGraph,
+    leaves: &[NodeIndex],
+    exit_node: Option<NodeIndex>,
+    tree_type: &TreeType,
+) -> Result<Vec<NodeIndex>, CompileError> {
+    let result = match kind {
+        TypedIntrinsicFunctionKind::SizeOfVal { exp } => connect_expression(
+            &(*exp).expression,
+            graph,
+            leaves,
+            exit_node,
+            "size_of",
+            tree_type,
+            exp.span.clone(),
+        )?,
+        TypedIntrinsicFunctionKind::GetPropertyOfType { .. } => {
+            let node = graph.add_node("Type Property".into());
+            for leaf in leaves {
+                graph.add_edge(*leaf, node, "".into());
+            }
+            vec![node]
+        }
+        TypedIntrinsicFunctionKind::GetStorageKey => {
+            let node = graph.add_node("Generate B256 Seed".into());
```

Comment:
```suggestion
            let node = graph.add_node("Get storage key".into());
```
I forgot to change this when I replaced `__generate_uid` with `__get_storage_key`.